rollbar-java/src/main/java/com/rollbar/notifier/Rollbar.java

```diff
@@ -612,7 +637,8 @@ private void process(Throwable error, Map<String, Object> custom, String descrip
     }
     
     // Pre filter
-    if (config.filter() != null && config.filter().preProcess(level, error, custom, description)) {
+    if (config.filter() != null && config.filter().preProcess(level, error.getThrowable(), custom,
+        description)) {
       LOGGER.debug("Pre-filtered error: {}", error);
```

Comment:
Here we used to debug log the throwable but now we will print the wrapper, not sure that matters.

Reply:
Good point! After looking at it i think it's ok because we provide a `toString()` method of the `ThrowableWrapper` implementation (https://github.com/rollbar/rollbar-java/blob/log4j2-integration/rollbar-java/src/main/java/com/rollbar/notifier/wrapper/RollbarThrowableWrapper.java#L81) and potentially the `Throwable` source could not be available.